.github/workflows/wheels.yml

```diff
@@ -69,35 +69,20 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
 
-      - name: Install cibuildwheel
+      - name: Build wheels
```

Comment:
Shall we keep install as its own step? Won't affect the timings, but helps fold away 40-odd lines of installation logs. 

The cibuildwheel logs are so long so may help to see the relevant stuff right away.

Reply:
Ok, I've pushed a commit, separating it for emulated, native and Windows.